internal/component/beyla/ebpf/beyla_linux.go

```diff
@@ -75,6 +77,60 @@ func (args Routes) Convert() *transform.RoutesConfig {
 	return routes
 }
 
+// Validate validates the sampler configuration
+func (args SamplerConfig) Validate() error {
+	if args.Name == "" {
+		return nil // Empty name is valid, will use default
+	}
+
+	validSamplers := map[string]bool{
+		"always_on":                true,
```

Comment:
Can we please make these an enum and compare as consts instead of strings directly?